EpikCord/client/http_client.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, token: Optional[str] = None, *args, **kwargs):
         self.session = ClientSession(
             *args,
             **kwargs,
-            json_serialize=lambda x, *__, **___: json.dumps(x).decode()
+            json_serialize=lambda x, *__, **___: json.dumps(x).decode() # type: ignore
```

Comment:
wth is this? xD

Reply:
Apparently it's not the right type of `json_serialize`, mainly because of the return type being specified or somethin, can't quite remember.